src/components/Month/getOptions.js

```diff
@@ -115,16 +128,20 @@ const renderDayEvents = (year, month, day, events) => {
     return currentEvents;
   });
 };
+
 const renderFullDay = (year, month, months, day, events, style) => {
   const hasEvents = dayHasEvents(year, month, day, events);
+  const isToday = dayIsToday(year, month, day);
   return (<Link
-    to={{ pathname: `/${months[month]}/${day}` }}
-    className={hasEvents ? `${styles[style]} ${styles.hasEvent}` : `${styles[style]}`}
+    to={{ pathname: `/${year}/${months[month]}/${day}` }}
+    className={hasEvents ?
+      `${styles[style]} ${styles.hasEvent} ${styles[isToday]}` :
```

Comment:
Use [classnames](https://github.com/JedWatson/classnames) library for concat classNames